stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/theme/StreamDimens.kt

```diff
@@ -93,7 +98,12 @@ public data class StreamDimens(
     public val userReactionItemIconSize: Dp,
     public val reactionOptionItemIconSize: Dp,
     public val headerElevation: Dp,
-    public val messageItemMaxWidth: Dp
+    public val messageItemMaxWidth: Dp,
+    public val quotedMessageTextVerticalPadding: Dp,
+    public val quotedMessageTextHorizontalPadding: Dp,
+    public val quotedMessageAttachmentPreviewSize: Dp,
+    public val quotedMessageAttachmentVerticalPadding: Dp,
+    public val quotedMessageAttachmentStartPadding: Dp
```

Comment:
Should we add `quotedMessageAttachmentEndPadding `padding here too if we have start padding?